server/use-console-timestamp.js

```diff
@@ -0,0 +1,21 @@
+const origLog = console.log.bind(console);
+const origWarn = console.warn.bind(console);
+const origErr = console.error.bind(console);
+
+const ts = () => {
+  return '[' + new Date().toISOString() + '] ';
+};
+
+const useConsoleTimestamp = () => {
+  console.log = function(message, ...optionalParams) {
+    origLog(ts() + message, ...optionalParams);
+  };
+  console.warn = function(...data) {
```

Comment:
How come `console.warn` and `console.error` have a different signature than `console.log`?

Reply:
addressed